draft-ietf-scitt-architecture.md

```diff
@@ -142,7 +142,7 @@ The rise in popularity of verifiable data structures as a mechanism to make acto
 
 This document defines a generic, interoperable and scalable architecture to enable transparency across any supply chain with minimum adoption barriers.
 It provides flexibility, enabling interoperability across different implementations of Transparency Services with various auditing and compliance requirements.
-Issuers can register their Signed Statements on any Transparency Service, with the guarantee that all Consumers will be able to verify them.
+Issuers can register their Signed Statements on any Transparency Service, with the guarantee that all Consumers[TODO: Fix Resolve Consumer] will be able to verify them.
```

Comment:
I'm flagging the term "Consumer" as we previously had Producers and Consumers. Tagging for an alternative term.